src/app/services/http-service.ts

```diff
@@ -54,7 +54,7 @@ export class HttpService extends Http {
       console.log('retryWhen callback');
       let count = 0;
       return attempts.flatMap(error => {
-        if (error.status == 0) { // Server offline :: keep trying
+        if (error.status == 0 && count < 3) { // Server offline :: keep trying
```

Comment:
How long do we try? Would it make sense to display "No internet connection. Try after sometime." after 10 or 100 attempts.